frontend/src/agenda/agenda.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div>
-    <div class="inline-title">{{ title }}</div>
+    <div class="q-inline-title">{{ title }}</div>
```

Comment:
just to explain what is going on here, I have created a base q.css file in src/assets/css and where/when I am using css classes that apply across multiple files i am placing them into this external css file.  In order to avoid name collisions and also to indicate where the css class is defined i am prefixing all such css classes with 'q-'